README.md

```diff
@@ -21,59 +21,58 @@ Once on a terminal window or command line, for simplicity, run commands from the
 
 ### Python
 
-Check for python installed. 
-This tool is [tested with Python 3.6, 3.7, and 3.8](https://app.travis-ci.com/dimagi/commcare-export).
+Check which Python version is installed.
 
+This tool is tested with Python versions from 3.8 to 3.12.
 
 ```shell
 $ python --version
 $ python3 --version
 ```
-If python is installed, all of its available versions would be listed.
+If Python is installed, its version will be listed.
 
-If python isn't installed, Install Python 3.8 from [this link](https://www.python.org/downloads/). 
+If Python isn't installed, [download and install](https://www.python.org/downloads/) the latest release. 
```

Comment:
should we say install any version from 3.8 to 3.12?
If the latest version is above 3.12, It could lead to unexpected errors?

Reply:
dff0651